src/components/Banner/Banner.stories.tsx

```diff
@@ -36,9 +36,8 @@ const getDescription = (status?: Variant) => (
       status="neutral"
       variant="link"
     >
-      click into the course
+      click into the course.
```

Comment:
I noticed that `VoicerOver` was reading out the period as it's own thing because it was outside of the link content, so I moved it in.